### PR TITLE
Remove uppercase filenames and links hdh 220

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,17 +20,17 @@
...
-    "@docusaurus/plugin-client-redirects": "3.0.1",
-    "@docusaurus/plugin-debug": "3.0.1",
-    "@docusaurus/plugin-google-analytics": "3.0.1",
-    "@docusaurus/plugin-google-gtag": "3.0.1",
-    "@docusaurus/plugin-sitemap": "3.0.1",
-    "@docusaurus/preset-classic": "3.0.1",
-    "@docusaurus/theme-classic": "3.0.1",
-    "@docusaurus/theme-mermaid": "3.0.1",
-    "@docusaurus/theme-search-algolia": "3.0.1",
+    "@docusaurus/core": "3.1.1",
```
Comment (SushrutHarness) [inline package.json:23]: Do we want to bump docusaurus here?

Comment (douglas-j-bothwell) [reply package.json:23]: @SushrutHarness 
No we don't. I need to remove this from the commit before merging. I need the updates temporarily to catch any links that are still broken due to case issues.

File: docs/platform/variables-and-expressions/runtime-inputs.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ helpdocs_category_id: fb16ljb8lu
 helpdocs_is_private: false
 helpdocs_is_published: true
 redirect_from:
-  - /docs/platform/References/runtime-inputs
+  - /docs/platform/references/runtime-inputs
```
Comment (aimurphy) [inline docs/platform/variables-and-expressions/runtime-inputs.md:10]: Are the redirects case sensitive?

File: yarn.lock [hunk 3]
```diff
@@ -1220,10 +1220,10 @@
     "@docsearch/css" "3.5.2"
     algoliasearch "^4.19.1"
 
-"@docusaurus/core@3.0.1":
-  version "3.0.1"
-  resolved "https://registry.yarnpkg.com/@docusaurus/core/-/core-3.0.1.tgz#ad9a66b20802ea81b25e65db75d4ca952eda7e01"
-  integrity sha512-CXrLpOnW+dJdSv8M5FAJ3JBwXtL6mhUWxFA8aS0ozK6jBG/wgxERk5uvH28fCeFxOGbAT9v1e9dOMo1X2IEVhQ==
+"@docusaurus/core@3.1.1":
```
Comment (aimurphy) [inline yarn.lock:1223]: I don't think we want to merge these yarn.lock changes yet. @ravilach ?

**Review by aimurphy — DISMISSED** [review]
Approved from CI perspective

File: yarn.lock (hunk 3 above)
Comment (douglas-j-bothwell) [reply yarn.lock:1223]: @aimurphy @ravilach I might have committed the yarn.lock changes by accident. I'm planning to revert yarn.lock and package.json after I fix all the links.

Comment (ravilach) [reply yarn.lock:1223]: Thanks @douglas-j-bothwell and @aimurphy not yet.

**Comment by jenniferopal** [comment]
There's a merge confict. Please resolve so I can review. My reviews tend to get dismissed once changes are made.

**Review by Debanitrkl — APPROVED** [review]
Looks good from IDP perspective.

**Review by brian-f-harness — APPROVED** [review]
LGTM

**Review by ravilach — APPROVED** [review]

**Pull request merged**